pkg/fleetautoscalers/fleetautoscalers_test.go

```diff
@@ -1950,7 +1951,7 @@ func TestApplyListPolicy(t *testing.T) {
 							}}}},
 			},
 			want: expected{
-				replicas: 0,
+				replicas: 1,
 				limited:  false,
```

Comment:
```suggestion
				limited:  true,
```

Per Mark's comment above this should be "true", so the logic will need to be modified in fleet_autoscalers.

Reply:
Please review the latest commit, which ensures that the replicas never scale down to zero. It also modifies the TestApplyListPolicy in fleetautoscaler_test.go to reflect this change.👍🏻